<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerrainEditDialog</class>
 <widget class="QWidget" name="TerrainEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QLineEdit" name="texture3"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="heightmap"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Heightmap</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="texture1"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Texture 1 (red)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Texture map</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="texture2"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Texture 2 (green)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButton_7">
     <property name="text">
      <string>New heightmap</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="textureMap"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Texture 3 (blue)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>TerrainEditDialog</receiver>
   <slot>onTex1Pick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>TerrainEditDialog</receiver>
   <slot>onTex2Pick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>TerrainEditDialog</receiver>
   <slot>onTex3Pick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>TerrainEditDialog</receiver>
   <slot>onTexMapPick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>TerrainEditDialog</receiver>
   <slot>onHeightmapPick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>TerrainEditDialog</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>TerrainEditDialog</receiver>
   <slot>onNewHeightmap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTex1Pick()</slot>
  <slot>onTex2Pick()</slot>
  <slot>onTex3Pick()</slot>
  <slot>onTexMapPick()</slot>
  <slot>onHeightmapPick()</slot>
  <slot>onSave()</slot>
  <slot>onNewHeightmap()</slot>
 </slots>
</ui>
